test/QueryPlannerTest.cpp

```diff
@@ -904,6 +904,7 @@ TEST(QueryExecutionTreeTest, testPoliticiansFriendWithScieManHatProj) {
     //                  "| width: 3} [0] with textLimit = 1 | width: 6} [0]\n) "
     //                  "| width: 6}",
     //              qet.asString());
+    /*
```

Comment:
Replace the commented part, `git` will remember the old version if we ever need it again